### PR TITLE
[Fix #5862] Update Firefox Accounts benefits page

### LINKED ISSUE #5862: FxA Benefits page v 1.1
For relaunch - Update the page based on learning from qualitative testing
Post relaunch - Compare GA data of relaunched page to the winner (new baseline) of the quantitative a/b test w. short vs. long copy

- [x] Design QA - Approval (by Lauren) - [QA document](https://docs.google.com/document/d/1UlSGIyekb82vBiAAhWdU0o_xSoYF3vQdIJ859dlFNYM/edit)
- [ ] Coded page final QA - Approval (by Lauren)

- Design file (sketch)
- [v 1.1 brief](https://docs.google.com/document/d/1wubD3fE6geu4jg_f9sgiMisf9TJ1pwfHEuwAYWz7_Js/edit#heading=h.da90bld12nbq)
- [Copy doc](https://docs.google.com/document/d/1G8QkQfQLfw4sILYK7JoYcIXLfh5c1F3DyLd3BMguewo/edit)

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Updates copy and layout, removes previous copy experiment, and adds new `button_text` and `button_class` params to the FxA button macro.

This is awaiting stakeholder review and final signoff so marking DNM for now.

This page is not yet localized so these string changes have no impact. I'm not sure if we'll open this to l10n at this point or if they want to continue tweaking. I'll find out.

## Issue / Bugzilla link
Issue #5862 

## Testing
https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/accounts/features/

### CODE REVIEW

**Comment by craigcook** [comment]
Confirmed this page is still subject to copy changes in the near future and remains English only for the time being. No l10n needed.

File: bedrock/base/templates/macros.html [hunk 1]
```diff
@@ -290,7 +290,7 @@ <h3>{{ _('Your download link was sent.') }}</h3>
   </section>
 {%- endmacro %}
 
-{% macro fxa_create_account_button(button_location, id_prefix, campaign, entrypoint, content = '', source = '' ) %}
+{% macro fxa_create_account_button(button_location, id_prefix, campaign, entrypoint, button_text, button_class, content = '', source = '') %}
```
Comment (stephaniehobson) [inline bedrock/base/templates/macros.html]: To make these parameters optional I think they need to be in the same format as content and source.

File: bedrock/base/templates/macros.html [hunk 2]
```diff
@@ -345,8 +353,8 @@ <h3>{{ _('Your download link was sent.') }}</h3>
     {{ download_firefox(dom_id=download_id, download_location=button_location) }}
   </div>
   <div class="show-fxa-supported-signed-out">
-    <a href="{{ accounts_url }}" class="button button-hollow button-light" data-button-name="Create account" data-link-type="button" data-cta-position="{{ button_location }}" id="{{ account_id }}">
-      {{ _('Create account') }}
+    <a href="{{ accounts_url }}" class="button {{ button_class }}" data-button-name="{{ button_text }}" data-link-type="button" data-cta-position="{{ button_location }}" id="{{ account_id }}">
```
Comment (stephaniehobson) [inline bedrock/base/templates/macros.html]: I think we should leave the data-button-name as "Create account". The current prod setup means that it's tracked as a create account button in GA no matter what locale it's in, I think we should maintain that.

File: bedrock/firefox/templates/firefox/accounts-features.html [hunk 3]
```diff
@@ -15,81 +15,62 @@
...
 {% block content %}
 <main>
     <header class="fab-head">
       <div class="content">
-        <h1 class="fab-head-title">{{_('Firefox Account')}}</h1>
-        <p class="fab-head-subtitle">{{_('One easy login.<br> Epic perks.')}}</p>
+        <h1 class="fab-head-title">{{_('Move from device to device easily with a Firefox Account.')}}</h1>
         <div class="fab-head-cta">
-          {{ fxa_create_account_button('primary cta', 'features-header', 'may2018', 'fxa.features', utm_content, 'account-features') }}
+          {{ fxa_create_account_button('primary cta', 'features-header', 'jul2018', 'fxa.features', _('Create an Account'), 'button-light', utm_content, 'account-features') }}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/accounts-features.html]: Did we get a request to change the campaign? The current parameter was provided by the accounts team and I wouldn't want to change it without consulting them.

File: bedrock/firefox/templates/firefox/accounts-features.html [hunk 4]
```diff
@@ -15,81 +15,62 @@
...
         </div>
       </div>
     </header>
 
-    <h2 class="fab-intro">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/accounts-features.html]: Without this `<h2>` all the perk headings need to move up a level.

File: bedrock/base/templates/macros.html (hunk 2 above)
Comment (craigcook) [reply bedrock/base/templates/macros.html]: Ah good point.

File: bedrock/firefox/templates/firefox/accounts-features.html (hunk 3 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/accounts-features.html]: I assumed, probably wrongly. I'll revert.

File: bedrock/firefox/templates/firefox/accounts-features.html (hunk 4 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/accounts-features.html]: Doh!

**Comment by craigcook** [comment]
Updated. Looks like the password image had some transparent pixels at the bottom. I've cropped it so it's nice and flush now.

**Comment by stephaniehobson** [comment]
GTG 🚥

**Pull request merged**